src/sentry/static/sentry/app/views/settings/components/settingsBreadcrumb/findFirstRouteWithoutRouteParam.jsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * For all routes with a `path`, find the first route without a route param (e.g. :apiKey)
```

Comment:
Doesn't this find the first route that doesn't start with a route param? It looks like it would find routes that have route params mid path.

Reply:
Ah, it was intended to make sure it didn't start with a route param, but I think it's probably better to check for any route params